packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -100,22 +108,359 @@ class VMInfoList extends StatelessWidget {
       ],
     );
   }
+}
+
+List<Widget> _prettyRows(BuildContext context, List<Row> rows) {
```

Comment:
Is there any reason these private methods were pulled out of `VMInfoList`?

Reply:
Because I used them on the RetainingPathWidget and the InboundReferencesWidget.